resolwe_bio/processes/differential_expression/deseq.yml

```diff
@@ -33,6 +33,15 @@
       type: list:data:expression
       description: |
         Control samples (replicates)
+    - name: options
+      label: Differential expression analysis options
+      group:
+        - name: beta_prior
+          label: betaPrior
```

Comment:
`Beta prior` would be nicer. You can keep the name of the parameter added in brackets, if you deem it important.